src/NexusMods.App.UI/Pages/Downloads/IInProgressViewModel.cs

```diff
@@ -30,6 +36,7 @@ public interface IInProgressViewModel : IPageViewModelInterface
     bool HasDownloads { get; }
 
     int ActiveDownloadCount { get; }
+    int CompletedDownloadCount { get; }
```

Comment:
This doesn't have to exist, it's just `CompletedTasks.Count` which is already reactive.